inspirehep/modules/workflows/workflows/core_selection.py

```diff
@@ -27,7 +27,7 @@ class CoreSelection(object):
 
     workflow = (
         halt_record(
-            action='auto_non_core_record',
+            action='hep_approval',
```

Comment:
oh wait this will not work, as we want to aggregate the workflows differently for `auto_non_core_records`